src/extensions/ThumbnailsExtension/ThumbnailsExtensionLoader.ts

```diff
@@ -17,8 +17,7 @@ export const ThumbnailsExtensionLoader = {
     }
 
     if (
-      config.thumbnails.src.substring(config.thumbnails.src.length - 4) !==
-      '.vtt'
+      ["vtt", "bif"].indexOf(config.thumbnails.src.split(".").pop()) < 0
```

Comment:
a bit hard to read imo, can we make this more explicit? Something like:

```javascript
const ext = ...;
if ( ext !== 'vtt' || ext !== 'bif' ) {
  return false;
}
```

Reply:
I agree. It has been updated [here](https://github.com/matvp91/indigo-player/compare/master...stoicbuddha:bifupdates?expand=1#diff-214c9a3b5a913f7953c1289b126d281bR18).